lib/racer-client.coffee

```diff
@@ -17,10 +17,14 @@ class RacerClient
       cb null
       return
 
+    editor_file_path = editor.getPath().substring(0, )
+    editor_folder_path = editor_file_path.substring(0, editor_file_path.lastIndexOf("/"))
```

Comment:
Also have you tested what happens if one is editing a new buffer which still hasn't been saved as a file but is in rust syntax?